providers/network/resources/http.go

```diff
@@ -67,9 +67,14 @@ func initHttpGet(runtime *plugin.Runtime, args map[string]*llx.RawData) (map[str
 			}
 		}
 
+		port := conn.Conf.Options["port"]
```

Comment:
at this point `conn.Conf.Port` should be set. This is what I meant by the previous comment I put here.